package.json

```diff
@@ -33,6 +33,7 @@
     "@babel/preset-env": "^7.5.5",
     "@babel/preset-react": "^7.0.0",
     "@babel/preset-typescript": "^7.3.3",
+    "@openmrs/esm-api": "1.0.3",
```

Comment:
`@openmrs/esm-api` 1.0.4 is broken, will work on figuring it out today

Reply:
How is it broken? Typescript?

Reply:
`openmrs-esm-api` can't find fhir.js, this is fixed by adding fhir.js in the dependencies for `openmrs-esm-api`